lib/controllers/foods-controller.js

```diff
@@ -8,7 +8,7 @@ function index (req, res) {
 };
 
 function create (req, res) {
-  Food.create(req)
+  Food.create(req.body.food.name, req.body.food.calories)
```

Comment:
🎉 It's super helpful to know about separation of concerns! @TheKaterTot 
Do you think we want to save these as constants and then pass them through, any idea on convention on whether or not to do that? #JSNoob